packages/babel-preset/__fixtures__/assets/output.ts

```diff
@@ -0,0 +1,13 @@
+import _asset from './blank.jpg';
+import { __styles } from '@griffel/react';
+import image from './blank.jpg';
+export const useStyles = __styles(
+  {
+    root: {
+      Bcmaq0h: 'fnwsaxv',
+    },
+  },
+  {
+    d: [`.fnwsaxv{background-image:url(${_asset});}`],
```

Comment:
I would also test the case where there are multiple assets, either a separate test or add more assets to this fixture

Reply:
There is a different test that is broken with Linaria v3, it includes multiple nested assets 🙃

I will modify this fixture to include more assets.

Reply:
Updated 5b6db17